docs/.vuepress/config.js

```diff
@@ -322,14 +297,8 @@ module.exports = {
                 '/project/repository-guide',
                 '/project/related-projects',
                 '/project/implementation-status',
-                ['https://github.com/ipfs/roadmap', 'Roadmap'],
                 ['https://github.com/ipfs/specs', 'Specifications'],
-                ['https://github.com/ipfs/research', 'Research'],
-                ['https://github.com/ipfs/team-mgmt', 'Team org planning'],
-                [
-                  'https://github.com/ipfs/community/blob/master/code-of-conduct.md',
-                  'Code of conduct'
-                ]
+                ['https://github.com/ipfs/notes', 'Research'],
```

Comment:
I think this should link to the docs research page rather than https://github.com/ipfs/notes